src/test/com/gd/twitterstreamingtToDatalake/TransformTweetsTest.scala

```diff
@@ -3,29 +3,20 @@ package com.gd.twitterstreamingtToDatalake
 import com.holdenkarau.spark.testing.StreamingSuiteBase
 import org.scalatest.{FunSuite, WordSpec}
 
-import scala.collection.mutable
-import org.apache.spark.rdd.RDD
-import org.apache.spark.streaming.dstream.InputDStream
-import org.apache.spark.streaming.{Seconds, StreamingContext}
 import twitter4j.Status
+import org.scalamock.scalatest.MockFactory
 
-class TransformTweetsTest extends FunSuite with StreamingSuiteBase {
+class TransformTweetsTest extends FunSuite with StreamingSuiteBase with MockFactory{
 
-  conf.setAppName("twitterStreaming").setMaster("local[2]")
+  val mockedStatus:Status = mock[MockStatus]
 
-  val ssc = new StreamingContext(conf, Seconds(5))
-
-  val lines: mutable.Queue[RDD[twitter4j.Status]] = mutable.Queue[RDD[twitter4j.Status]]()
-
-  val dstream: InputDStream[Status] = ssc.queueStream(lines)
-  lines += sc.makeRDD(Seq(twitter4j.Status,"To be or #not to be." ,"That is the #question."))
+  var input = new List[mockedStatus]
```

Comment:
> when I tried to mock twitter4j.Status, there was this compiler error which was not getting the mockedStatus.

That's because the way you are trying to apply mocking is incorrect. 
Please, learn how to do it properly (e.g. http://scalamock.org/quick-start/).